index.bs

```diff
@@ -5041,8 +5047,8 @@ function process(numberOfFrames) {
 	} // End of render quantum loop
 
 	if (currentTime >= stop) {
-		// end playback state of this node.
-		// no further invocations of process() will occur.
+		// End playback state of this node.  No further invocations of process()
+		// will occur.  Set the number of output channels to 1.
```

Comment:
To make it clear that output channel count is not set to 1 immediately, perhaps
drop the last sentence and use something like "will occur, as output for
future rendering quanta will contain only a single channel of silence."

Reply:
Chrome can't guarantee that this happens exactly on the rendering quantum after stop.  It may take several renders depending on how all he locks line up.

Reply:
Perhaps something like "Schedule a change in the number of output channels to 1."